AntennaTracker/system.cpp

```diff
@@ -92,6 +92,7 @@ void Tracker::init_ardupilot()
         // for some servos)
         prepare_servos();
     }
+    AP_Notify::flags.initialised = true;
```

Comment:
maybe hal.scheduler->setup_complete() ?
or rename hal.scheduler->system_initialized() to hal.scheduler->set_system_initialized()

Reply:
Fixed